cdap-ui/app/cdap/components/FieldLevelLineage/v2/LineageSummary/index.tsx

```diff
@@ -28,40 +31,152 @@ const styles = (theme) => {
       display: 'flex',
       justifyContent: 'space-between',
     },
+    container: {
+      position: 'absolute' as 'absolute',
+      height: '100%',
+      width: '100%',
+    },
   };
 };
 
-function LineageSummary({ classes }) {
-  return (
-    <Consumer>
-      {({ causeSets, target, targetFields, impactSets, firstCause, firstImpact, firstField }) => {
-        return (
-          <div className={classes.root}>
-            <div>
-              <FllHeader type="cause" first={firstCause} total={Object.keys(causeSets).length} />
-              {Object.keys(causeSets).map((key) => {
-                return <FllTable key={key} tableName={key} fields={causeSets[key]} />;
-              })}
-            </div>
-            <div>
-              <FllHeader
-                type="target"
-                first={firstField}
-                total={Object.keys(targetFields).length}
-              />
-              <FllTable key={target} tableName={target} fields={targetFields} />
-            </div>
-            <div>
-              <FllHeader type="impact" first={firstImpact} total={Object.keys(impactSets).length} />
-              {Object.keys(impactSets).map((key) => {
-                return <FllTable key={key} tableName={key} fields={impactSets[key]} />;
-              })}
+class LineageSummary extends React.Component<{ classes }> {
+  private activeLinks;
+
+  // TO DO: Get colors from theme once we've created a separate theme colors file
```

Comment:
nit: Not in this PR but in the next one remove this comment. Don't think its valid anymore.